eclass/qt6-build.eclass

```diff
@@ -133,48 +131,48 @@ qt6-build_src_install() {
 # @DESCRIPTION:
 # <flag> is the name of a flag in IUSE.
 qt_feature() {
-	[[ $# -ge 1 ]] || die "${FUNCNAME}() requires at least one argument"
-	echo "-DQT_FEATURE_${2:-$1}=$(usex $1 ON OFF)"
+	[[ ${#} -ge 1 ]] || die "${FUNCNAME}() requires at least one argument"
+
+	echo "-DQT_FEATURE_${2:-${1}}=$(usex ${1} ON OFF)"
 }
 
 # @FUNCTION: qt6_symlink_binary_to_path
 # @USAGE: <target binary name> [suffix]
 # @DESCRIPTION:
-# Symlink a given binary from QT6_BINDIR to QT6_PREFIX/bin, with optional suffix
+# Symlink a given binary from QT6_BINDIR to QT6_PREFIX/bin, with
+# optional suffix.
 qt6_symlink_binary_to_path() {
-	[[ $# -ge 1 ]] || die "${FUNCNAME}() requires at least one argument"
+	[[ ${#} -ge 1 ]] || die "${FUNCNAME}() requires at least one argument"
```

Comment:
I don't really see the point of these stylistic changes, but whatever...

Reply:
For consistency, the eclass already had a mix of both, e.g.
`dosym -r "${QT6_BINDIR}"/${1} /usr/bin/${1}${2}`
I don't mind either but I prefer it's all the same within one eclass/ebuild. Typically put ${} unconditionally, so picked that one.